Tests/FoundationTests/FileManagerExtensionsTests.swift

```diff
@@ -80,6 +80,30 @@ final class FileManagerExtensionsTests: XCTestCase {
         } catch {}
     }
 
+    func testEncodeJSON() {
+        let point = CGPoint(x: 1, y: 2) //CGPoint conforms to Encodable
+        do {
+            let temporaryDirectoryURL = FileManager.default.temporaryDirectory
+            let temporaryFilename = ProcessInfo().globallyUniqueString
+            let temporaryFileURL = temporaryDirectoryURL.appendingPathComponent(temporaryFilename)
+
+            XCTAssertTrue(try FileManager.default.encodeJSON(point, to: temporaryFileURL))
+            XCTAssert(FileManager.default.fileExists(atPath: temporaryFileURL.path))
+
+            let data = FileManager.default.contents(atPath: temporaryFileURL.path)
+            XCTAssertNotNil(data)
+
+            let decoder = JSONDecoder()
+            let decodedPoint = try decoder.decode(CGPoint.self, from: data!)
+            XCTAssertTrue(point == decodedPoint)
+
+            try FileManager.default.removeItem(at: temporaryFileURL)
+            XCTAssertFalse(FileManager.default.fileExists(atPath: temporaryFileURL.path))
```

Comment:
You don't really need this check, you can actually change the line above to:
```Swift
XCTAssertTrue(try FileManager.default.removeItem(at: temporaryFileURL))
```
Even that you don't really need, because if the removal fails it will throw an exception, which you will catch.
Sorry that I didn't catch this in the last review... This one seems to be going on forever 😅